server/routes/api/auth.js

```diff
@@ -9,26 +13,55 @@ const User = require('../../models/user');
 router.post('/login', (req, res) => {
   User.authenticate(req.body.email, req.body.password)
     .then(
-      user => {
+      (user) => {
         req.session.userId = user._id;
```

Comment:
ESLint does not like having a object property with `_` in front of the name of the property. Disabled Rule.